### PR TITLE
ipq40xx: convert Cell C RTL30VW to DSA

### PR DESCRIPTION
**Pull request by PolynomialDivision**
**Only compile tested**

Person who added device: @CHKDSK88
Please test

### CODE REVIEW

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-rtl30vw.dts [hunk 1]
```diff
@@ -386,3 +386,27 @@
...
+};
+
+&swport4 {
+	status = "okay";
+
+	label = "lan1";
+};
+
+&swport5 {
+	status = "okay";
```
Comment (CHKDSK88) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-rtl30vw.dts]: Why it's makred as okay? Cell has only two ports.

**Comment by CHKDSK88** [comment]
@PolynomialDivision 

It works, but please remove wan port.

**Comment by PolynomialDivision** [comment]
@CHKDSK88 Can you test again? I deleted now `swport5`. Just to be sure.

**Comment by CHKDSK88** [comment]
@PolynomialDivision 
Everything work as expected.
`Tested-by: Pawel Dembicki <paweldembicki@gmail.com>`

**Comment by PolynomialDivision** [comment]
Thanks a lot :D

**Comment by PolynomialDivision** [comment]
ping @robimarko

**Comment by robimarko** [comment]
What about MAC-s?
I picked this, but will leave it open until MAC setting is confirmed

**Comment by CHKDSK88** [comment]
It's the same as non dsa version.

**Comment by robimarko** [comment]
Ok, then thanks.

**Pull request closed without merging**